libc/src/__support/str_to_float.h

```diff
@@ -150,9 +111,7 @@ eisel_lemire(ExpandedFloat<T> init_num,
   // accuracy, and the most significant bit is ignored.) = 9 bits. Similarly,
   // it's 6 bits for floats in this case.
   const uint64_t halfway_constant =
-      (uint64_t(1) << (BITS_IN_MANTISSA -
-                       fputil::FloatProperties<T>::MANTISSA_WIDTH - 3)) -
-      1;
+      (uint64_t(1) << (BITS_IN_MANTISSA - FloatProp::MANTISSA_WIDTH - 3)) - 1;
```

Comment:
given the other places where this is changed to `BITS_IN_MANTISSA - (FloatProp::MANTISSA_WIDTH + 3)` this should probably be changed to match.